code/modules/power/singularity/singularity.dm

```diff
@@ -629,6 +666,8 @@
 */ //Fuck you centcomm
 
 /obj/machinery/singularity/Destroy()
+	consume(555)
```

Comment:
What is this for?

Reply:
Hm, probably left over testing from when the supersea was eating my singulos. I'll rip it out.